website/src/app/kb/client-apps/linux-gui-client/readme.mdx

```diff
@@ -158,7 +158,14 @@ captured by systemd and sent to journald. To view the logs of the IPC service,
 use:
 
 ```bash
-journalctl -efu firezone-client-ipc.service
+journalctl --pager-end --follow --unit firezone-client-ipc.service
+```
+
+The GUI client logs to journald directly as well with the syslog identifier
+`firezone-client-gui`. To view the logs via `journalctl`, use:
+
+```bash
+journalctl --pager-end --follow --identifier firezone-client-gui
```

Comment:
To make the difference between `--unit` and `--identifier` more apparent, I used the long form of all options in these snippets. A Linux-versed admin can use the short form if they want to.

Reply:
👍